host/sgx/windows/sgxquoteexloader.c

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <Windows.h>
+#include <openenclave/internal/defs.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/trace.h>
+#include <stdlib.h>
+#include <string.h>
+#include "../../../common/oe_host_stdlib.h"
+#include "../sgxquote_ex.h"
+
+static const char* _quote_ex_library_file_name = "libsgx_quote_ex.dll";
+
+static oe_sgx_quote_ex_library_t* _quote_ex_library = NULL;
+
+static void _unload_quote_ex_library(void)
+{
+    if (_quote_ex_library && _quote_ex_library->handle)
+    {
+        OE_TRACE_INFO(
+            "_unload_quote_ex_library() %s\n", _quote_ex_library_file_name);
+        FreeLibrary((HMODULE)_quote_ex_library->handle);
+        _quote_ex_library->handle = 0;
+        if (_quote_ex_library->mapped)
+        {
+            oe_free(_quote_ex_library->mapped);
+            _quote_ex_library->mapped = NULL;
+        }
+        if (_quote_ex_library->uuid)
+        {
+            oe_free(_quote_ex_library->uuid);
+            _quote_ex_library->uuid = NULL;
+        }
+        if (_quote_ex_library->sgx_key_id)
+        {
+            oe_free(_quote_ex_library->sgx_key_id);
+            _quote_ex_library->sgx_key_id = NULL;
+        }
+    }
+}
+
+void oe_sgx_load_quote_ex_library(oe_sgx_quote_ex_library_t* library)
+{
+    oe_result_t result = OE_UNEXPECTED;
+
+    if (!library)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    _quote_ex_library = library;
+
+    if (library->handle == 0)
+    {
+        HMODULE _handle = 0;
```

Comment:
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says:
> The single underscore (_ ) is reserved for local definitions (static, file-scope definitions). e.g. static oe_result_t _parse_sgx_report_body(..).

So this variable should not have an underscore prefix since it is not static

Reply:
The variable itself is indeed a static one. It is pointer assigned to the address passed the function as a parameter.